### DISCUSSION TITLE
Cannot set Super+[ / Super+] as next/previous tab on Mac OS

### DISCUSSION THREAD

**Discussion by bitboxer** [ghostty-org/ghostty#3780] [Feature Requests, Ideas] [answered]
I added

```
keybind = all:super+shift+[=goto_split:previous
keybind = all:super+shift+]=goto_split:next
keybind = super+]=next_tab
keybind = super+[=previous_tab
```

to change the next/previous tab shortcut, but sadly it is not working, those lines do not change the next/previous tab shortcut and they stay at the default.

**Comment by mitchellh** [comment, marked as answer]
Duplicate, search for overriding system bindings.

**Reply by bitboxer** [reply]
Thank you. Found it.

**Reply by 00-kat** [reply]
For any others who find it, the relevant discussion is #3187.

**Reply by esn89** [reply]
Any easy way to find out what type of system setting is hijacking it?

I have a similar issue where super+shift>h isn't working.
